common/src/main/java/com/wynntils/commands/wynntils/WynntilsConfigCommand.java

```diff
@@ -30,12 +30,14 @@
 import net.minecraft.network.chat.TextComponent;
 import org.apache.commons.lang3.StringUtils;
 
-public abstract class WynntilsConfigCommand {
-    public static final SuggestionProvider<CommandSourceStack> featureSuggestionProvider =
+public final class WynntilsConfigCommand {
+    private WynntilsConfigCommand() {}
```

Comment:
Why?

Reply:
(talking about every empty constructor)

Reply:
I think the goal is to avoid unsubclassed abstract classes, which is done with the final + private constructor.

Reply:
Yeah but this is kinda annoying and messy?

Reply:
As @Incompleteusern says, it is to disallow instantiation. If you think it is too messy, I can remove it.

Reply:
(I agree that it is a bit messy, but ut also makes it clear it is a utility class. I'm on the fence for that one)

Reply:
This one kinda bothers me? What about @P0keDev @Incompleteusern @HighCrit ?

Reply:
If it bothers you, I'll remove it.

Reply:
In the past a decision was made to not have empty constructors on utility classes. When I proposed an empty constructor or one that throw an exception or both.

Reply:
I'm favour of preventing instantiation on utility classes. But I won't lay awake on it. Wynntils isn't really used as a dependency anywhere and we're all capable enough to know or notice it in PRs.

Reply:
I just removed them, so, hey... :) I think they have some value, but as HighCrit says it also unlikely that we will make any mistakes regarding this, and they do add a bit of clutter. So let's avoid the clutter, then.